src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs

```diff
@@ -29,12 +30,14 @@ namespace Serilog.Sinks.Raygun
     /// </summary>
     public class RaygunSink : ILogEventSink
     {
-        readonly IFormatProvider _formatProvider;
-        readonly string _userNameProperty;
-        readonly string _applicationVersionProperty;
-        readonly IEnumerable<string> _tags;
-        readonly IEnumerable<string> _ignoredFormFieldNames;
-        readonly RaygunClient _client;
+        private readonly IFormatProvider _formatProvider;
```

Comment:
Nit: the Serilog codebase doesn't use explicit `private` or `internal` modifiers.